raster/src/main/scala/geotrellis/raster/triangulation/DelaunayRasterizer.scala

```diff
@@ -0,0 +1,155 @@
+package geotrellis.raster.triangulation
+
+import com.vividsolutions.jts.geom.Coordinate
+
+import geotrellis.raster._
+import geotrellis.vector._
+import geotrellis.vector.triangulation._
+
+object DelaunayRasterizer {
+
+  /**
+   * Produces a Tile with shape given by a RasterExtent where pixels are linearly interpolated
+   * values of the z-coordinates of each triangle vertex.
+   */
+  def rasterize(
```

Comment:
Probably worth noting that this rasterizer is pretty specialized.  A more general interface would be desirable.